tune_sklearn/tune_gridsearch.py

```diff
@@ -70,7 +73,7 @@ class TuneGridSearchCV(TuneBaseSearchCV):
             For integer/None inputs, if the estimator is a classifier and ``y``
             is either binary or multiclass, :class:`StratifiedKFold` is used.
             In all other cases, :class:`KFold` is used. Defaults to None.
-        refit (bool, str, or `callable`):
+        refit (bool or str):
```

Comment:
Callable is still correct here right?

Reply:
The only thing I changed with refit here is that it has to be the name of the scorer you want to use to evaluate which hyperparameters are the best when you do multimetric. But I was looking through the code and I didn't see any part where a callable could be used for refit. I think having callable here in the docs might just be a mistake?